CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## master (unreleased)
 
 ### New Features
-
+- dynamic: add support for VMRay dynamic sandbox traces #2208 @mike-hunhoff @r-sm2024 @mr-tz
```

Comment:
maybe consider adding a few sentences to the release notes to explain this exciting new feature?

Reply:
https://github.com/mandiant/capa/pull/2208/commits/5afea2947394382f014a101f84d53ea3b8f26169